src/Sulu/Bundle/MediaBundle/Media/FormatCache/FormatCacheClearer.php

```diff
@@ -19,10 +19,18 @@
 class FormatCacheClearer implements FormatCacheClearerInterface
 {
     /**
-     * @var FormatCacheInterface[]
+     * @var array<string, FormatCacheInterface>
      */
     private $caches = [];
 
+    /**
+     * @param iterable<string, FormatCacheInterface> $caches
+     */
+    public function __construct(iterable $caches = [])
+    {
+        $this->caches = [...$caches];
+    }
+
```

Comment:
This is always an empty array on this branch or I'm missing something?

Reply:
Yes this is correct because otherwise it would have the cache clearer twice once from the constructor and once from the `add` calls. In 3.0 this will be non empty.